src/node-backend/controller/classroom-controller.js

```diff
@@ -11,12 +11,17 @@ export async function sendStudentCodesPerEmail(classId) {
 
 
 export async function getJoiningStudents(classId) {
-
+    return dbAdapter.getStudentCodes(classId);
 }
 
 
 export async function createClassroom(username, classname) {
-
+    const queryResult = await dbAdapter.createClassroom(username, classname);
+    if (queryResult['warningStatus'] === 0) {
+        return Number(queryResult['insertId']);
```

Comment:
what's insertId for?

Reply:
it's the ID that is created when the row is inserted into the table, which is the classId that is returned